internal/util/util.go

```diff
@@ -36,26 +36,16 @@ const (
 	BtPlaceStatsVarPrefix = "d/0/"
 	// BtStatVarGroup is the key for statvar group cache.
 	BtStatVarGroup = "d/1"
-	// BtPopObsPrefix for internal pop obs cache.
-	BtPopObsPrefix = "d/2/"
```

Comment:
@pradh We could also remove these cache from prophet

Reply:
d/2/ is needed for pop-obs validation pipeline during import, until we switch import flows

so we're not using this obsid->{observation-about, variable-measured} cache?

Reply:
Not any more, observation properties are fetched using BQ